live-build/hooks/30-fix-timedatectl.chroot

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+mv /usr/bin/timedatectl /usr/bin/timedatectl.real
+
+cat >/usr/bin/timedatectl<<EOF
+#!/bin/sh
+
+# wrapper to make sure timedatectl works with the
+# writable-paths setup on ubuntu core (making the symlink
+# absolute instead of using a relative path)
+# LP: #1650688 for details
+
+set -e
+
+TIMEDATECTL=/usr/bin/timedatectl.real
+
+case \$1 in
```

Comment:
Why do you quote everything instead of using:
```
cat > foo <<'EOF'
No substitution of $1 or $2 or anything with 'EOF'
EOF
```
? It seems the 'EOF' version is easier to read, no?